### PR TITLE
Server only feedback

### PR DESCRIPTION
**Pull request by BryceStevenWilley**
Another branch that I made and didn't merge!

This enables more feedback sessions to stay on the server, even if folks don't want to share their answers (things were hardcoded to assume that before).

I'd like to test a bit before merging, but feel free to try it out yourself!

### CODE REVIEW

File: docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml [hunk 1]
```diff
@@ -1,54 +1,80 @@
...
+  <h2 class="h4">Reviews</h2>
+
+  ${ collapse_template(what_are_reviews_template) }
 
-  * count: ${ review_agg['count'] }
-  * average: ${ str(round(review_agg['average'] * 1000)/1000) }
+  Interview file | Version | Number of reviews | Average Score
+  -------------- |---------|-------------------|---------------
+  % for review_agg in get_good_or_bad():
+  ${ review_agg['interview'] } | ${ review_agg['version'] } | ${ review_agg['count'] } | ${ str(round(review_agg['average'] * 1000)/1000) }
```
Comment (nonprofittechy) [inline docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml:44]: Cool! I like the idea of separating reviews by version

File: docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml [hunk 2]
```diff
@@ -60,20 +86,25 @@ if: ids
 id: main browse screen
 question: |
   Select feedback to browse
+subquestion: |
+  There are two main types of feedback:
+
+  * feedback attached to Github Issues
+  * feedback not attached to Github Issues (Reviews)
 fields:
   - What type of feedback: feedback_type
```
Comment (nonprofittechy) [inline docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml:95]: This is still pretty clunky but I know it's out of scope for this PR.

**Review by nonprofittechy — APPROVED** [review]

File: docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml (hunk 2 above)
Comment (BryceStevenWilley) [reply docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml:95]: Totally fair: I'll try to revive https://github.com/SuffolkLITLab/docassemble-GithubFeedbackForm/issues/28 with the progress so far after merging to discuss what we should change and how.

Comment (nonprofittechy) [reply docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml:95]: I'm thinking more that we show all kinds of feedback on one screen; the mode switching is not super intuitive. But it's possible we need modes; if that's the case, maybe a section/tab metaphor would be better.

Comment (BryceStevenWilley) [reply docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml]: Thanks. TBH, I was getting confused when trying to test this; how I split it up (and the names, ugh) weren't great. I agree, one screen makes a lot more sense, so I cleaned up a bit of code to do that.

At some point, there'll need to be some way to handle lots of feedback, but we'd run into usage issues (there are 100s of pieces of feedback on this server, I can't find new ones!) before we run into performance issues, and the old way of doing things wasn't going to deal with the usage issues either.

**Comment by BryceStevenWilley** [comment]
Cleaned up the feedback browser a bit more to show things on fewer screens if you want to look at it again, I'll merge sometime tomorrow either way.

**Comment by nonprofittechy** [comment]
> Cleaned up the feedback browser a bit more to show things on fewer screens if you want to look at it again, I'll merge sometime tomorrow either way.

Looks good to me!

**Pull request merged**